packages/web/src/renderHTML.ts

```diff
@@ -66,8 +67,9 @@ export default function renderHTML(config, manifest, manifestFiles, req, res) {
 
   try {
     webPage = cache.get('template');
+    const needHtmlCache = _.get(config, 'web.cache', undefined) || _.get(config, 'web.cache', true);
```

Comment:
I would recommend that [optional chaining]() be used here instead of `lodash` to avoid the `import`.
```suggestion
    const needHtmlCache = [undefined, null].includes(config?.web?.html_cache) ? true : config.web.html_cache
```